shipctl/x86_64/WindowsServer_2016/utility.ps1

```diff
@@ -616,9 +631,22 @@ function notify() {
   if (!$env:opt_username) {
     $opt_username = $env:NOTIFY_USERNAME
     if (!$env:opt_username) {
-      $env:opt_username = "Shippable"
+      if ($r_method -eq "irc") {
+        $env:opt_username = "Shippable-$env:BUILD_NUMBER"
```

Comment:
I didn't understand this bit. Why does the IRC user name have to contain the build number?

Reply:
it's a precaution in case you have 2 builds in parallel trying to post to irc. one would possibly hit a duplicate nick error.  User can always supply their own if they don't like the fact that it changes.